### DISCUSSION TITLE
Support SVG logo in pdf

### DISCUSSION THREAD

**Discussion by jf-viguier** [PrestaShop/PrestaShop#41079] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

Shop logo in SVG is not supported in pdf (invoice, ...)

### Describe the solution you'd like

I guess that TCPDF can include svg directly in pdf format. So there is no need to block svg format as decided by @marionf here https://github.com/PrestaShop/PrestaShop/pull/23959#issuecomment-827623447

**Comment by jf-viguier** [comment]
TCPDF example here : https://tcpdf.org/examples/example_058/

**Comment by hibatallahAouadni** [comment]
Hello @jf-viguier 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you
